secant/UIComponents/DesignGuide.swift

```diff
@@ -189,3 +189,31 @@ struct DesignGuide_Previews: PreviewProvider {
         .previewLayout(.fixed(width: 1086, height: 1080))
     }
 }
+
+// MARK: - Pressed Simulated
+
+extension Button {
+    var primaryButtonPressedStyle: some View {
+        buttonStyle(
+            StandardButtonStyle(
+                foregroundColor: Asset.Colors.Text.button.color,
+                background: Asset.Colors.Buttons.primaryButtonPressed.color,
+                pressedBackgroundColor: Asset.Colors.Buttons.primaryButtonPressed.color,
+                disabledBackgroundColor: Asset.Colors.Buttons.primaryButtonDisabled.color
+            )
+        )
+    }
+    
+    var activeButtonPressedStyle: some View {
+        buttonStyle(
+            StandardButtonStyle(
+                foregroundColor: Asset.Colors.Text.activeButtonText.color,
+                background: Asset.Colors.Buttons.activeButtonPressed.color,
+                pressedBackgroundColor: Asset.Colors.Buttons.activeButtonPressed.color,
+                disabledBackgroundColor: Asset.Colors.Buttons.activeButtonDisabled.color
+            )
+        )
+    }
+}
+
+
```

Comment:
[non-blocking | nit | typo] extra blankspace at end of file

```suggestion
```

Reply:
again, updated it on my side